piet-cairo/src/lib.rs

```diff
@@ -499,3 +519,20 @@ fn compute_blurred_rect(rect: Rect, radius: f64) -> (ImageSurface, Point) {
     let origin = rect_exp.origin();
     (image, origin)
 }
+
+fn write_rgba(data: &mut [u8], offset: usize, x: usize, r: u8, g: u8, b: u8, a: u8) {
+    // From the cairo docs for CAIRO_FORMAT_ARGB32:
+    //  each pixel is a 32-bit quantity, with alpha in the upper 8 bits, then red,
+    //  then green, then blue. The 32-bit quantities are stored native-endian.
```

Comment:
Bit of a nitpick, but why 2 spaces on all subsequent lines? 1 is used everywhere else.

Reply:
Well... it was meant as indication that it is a quote. I now changed it to `> ` instead.